app/src/js/components/Rules/add.js

```diff
@@ -124,8 +110,8 @@ const AddRule = ({
 
   useEffect(() => {
     setEnums({
-      collection: collections.list.data.map(getCollectionId).sort(asc),
-      provider: providers.list.data.map(({ id }) => id).sort(asc),
+      collection: get(collections, 'dropdowns.collectionName.options', []).map(({ id }) => id).sort(asc),
```

Comment:
Just to be sure I'm understanding, the `dropdowns` data has `id`? It looks like the `getOptionsCollectionName` query would only return `name,version` but I'm unsure I'm understanding correctly.